packages/react-spinner/src/components/Spinner/useSpinnerStyles.ts

```diff
@@ -1,34 +1,336 @@
+      cy: '50%',
+      fill: 'none',
+    },
+  },
+
+  tiny: {
+    [`& > svg`]: {
+      height: spinnnerSizes.tiny,
+      width: spinnnerSizes.tiny,
+    },
+    [`& > svg > circle`]: {
+      strokeWidth: spinnerStrokeWidth.sWidth,
+      r: rvalues.tiny,
+    },
+  },
+
+  extraSmall: {
+    [`& > svg`]: {
+      height: spinnnerSizes.extraSmall,
+      width: spinnnerSizes.extraSmall,
+    },
+    [`& > svg > circle`]: {
+      strokeWidth: spinnerStrokeWidth.sWidth,
+      r: rvalues.extraSmall,
+    },
+  },
+
+  small: {
+    [`& > svg`]: {
+      height: spinnnerSizes.small,
+      width: spinnnerSizes.small,
+    },
+    [`& > svg > circle`]: {
+      strokeWidth: spinnerStrokeWidth.sWidth,
+      r: rvalues.small,
+    },
+  },
+
+  medium: {
+    [`& > svg`]: {
+      height: spinnnerSizes.medium,
+      width: spinnnerSizes.medium,
+    },
+    [`& > svg > circle`]: {
+      strokeWidth: spinnerStrokeWidth.mWidth,
+      r: rvalues.medium,
+    },
   },
 
-  // TODO add additional classes for different states and/or slots
+  large: {
+    [`& > svg`]: {
+      height: spinnnerSizes.large,
+      width: spinnnerSizes.large,
+    },
+    [`& > svg > circle`]: {
+      strokeWidth: spinnerStrokeWidth.mWidth,
+      r: rvalues.large,
+    },
+  },
+
+  extraLarge: {
+    [`& > svg`]: {
+      height: spinnnerSizes.extraLarge,
+      width: spinnnerSizes.extraLarge,
+    },
+    [`& > svg > circle`]: {
+      strokeWidth: spinnerStrokeWidth.mWidth,
+      r: rvalues.extraLarge,
+    },
+  },
+
+  huge: {
+    [`& > svg`]: {
+      height: spinnnerSizes.huge,
+      width: spinnnerSizes.huge,
+    },
+    [`& > svg > circle`]: {
+      strokeWidth: spinnerStrokeWidth.lWidth,
+      r: rvalues.huge,
+    },
+  },
+
+  // global class for Spinner track
+  spinnerTrack: {
+    [`& > svg > circle.fui-spinner__Track`]: {
+      stroke: tokens.colorNeutralBackground4,
+    },
+  },
+
+  // modifier class for Spinner track if appearance="inverted"
+  spinnerTrackInverted: {
+    [`& > svg > circle.fui-spinner__Track`]: {
+      stroke: tokens.colorNeutralBackgroundInverted,
+    },
+  },
+
+  // global Spinner trail class
+  spinnerTail: {
+    [`& > svg > circle.fui-spinner__Tail`]: {
+      stroke: tokens.colorBrandStroke1,
+      animationName: {
+        '0%': {
+          strokeDasharray: '1,150',
+          strokeDashoffset: '0',
+        },
+
+        '50%': {
+          strokeDasharray: '90,150',
+          strokeDashoffset: '-35',
+        },
+
+        '100%': {
+          strokeDasharray: '90,150',
+          strokeDashoffset: '-124',
+        },
+      },
+      animationDuration: '1.5s',
+      animationIterationCount: 'infinite',
+      animationTimingFunction: 'cubic-bezier(0.33,0,0.67,1)',
+      strokeLinecap: 'round',
+      transform: 'rotate(-90deg)',
+      transformOrigin: '50% 50%',
+    },
+  },
+
+  // modifier class for Spinner trail
+  spinnerTailInverted: {
+    [`& > svg > circle.fui-spinner__Tail`]: {
```

Comment:
You should use regular single quote instead of string template quotes unless you are injecting a value.